src/vulkan/device.cc

```diff
@@ -547,6 +574,26 @@ Result Device::Initialize(
   ptrs_.vkGetPhysicalDeviceMemoryProperties(physical_device_,
                                             &physical_memory_properties_);
 
+  subgroup_size_control_properties_ = {};
+  if (use_physical_device_features_2 &&
+      std::find(
```

Comment:
Please move the find outside of the if conditional.

Reply:
done